src/components/RunningTrackComponent.vue

```diff
@@ -199,7 +199,12 @@ export default defineComponent({
       }
     },
     sortedDrawingOrder(teams: Array<Team> | undefined) {
-      return [...teams || []].sort((a, b) => a.rounds - b.rounds);
+      return [...teams || []].sort((a, b) => {
+        const diff = a.rounds - b.rounds
+        if (diff !== 0) return diff;
+
+        return a.updatedAt - b.updatedAt;
+      });
```

Comment:
`progress` is number between 0 and 1, so we can just sum it with the current rounds.
```suggestion
      return [...teams || []].sort((a, b) => (a.rounds + a.progress) - (b.rounds + b.progress));
```